Week-2/Session 5/Sleep recommendation list (CSS)/recommend.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>Sleep Duration Recommendations</title>
+   
+   <style>
```

Comment:
This is okay for now, But its always better to make a seperate css file to add the CSS. So that you can re-use the classes

Reply:
Ok, sure I will keep that in practice.